cmd/clusterctl/client/get_kubeconfig_test.go

```diff
@@ -63,7 +120,62 @@ func Test_clusterctlClient_GetKubeconfig(t *testing.T) {
 				return
 			}
 			g.Expect(err).ToNot(HaveOccurred())
-			g.Expect(config).ToNot(BeEmpty())
+			g.Expect(config).To(BeEquivalentTo(tt.want))
```

Comment:
```suggestion
			g.Expect(config).To(Equal(tt.want))
```
since actual and expected are both of the same type (string) we can use the `Equal` matcher here.